region/brinstar/kraid.json

```diff
@@ -1211,17 +1289,54 @@
                     {
                       "id": "A",
                       "requires": [
-                        {"enemyKill":{
-                          "enemies": [
-                            [
-                              "Green Space Pirate (standing)", "Green Space Pirate (standing)", "Green Space Pirate (standing)"
-                            ],
-                            [ "Mini-Kraid" ]
-                          ]
-                        }}
+                        {"or": [
+                          {"enemyKill":{
+                            "enemies": [["Green Space Pirate (standing)", "Green Space Pirate (standing)", "Green Space Pirate (standing)"]],
+                            "excludedWeapons": ["Bombs"]
+                          }},
+                          {"and": [
+                            {"enemyKill": {
+                              "enemies": [["Green Space Pirate (standing)", "Green Space Pirate (standing)", "Green Space Pirate (standing)"]]
```

Comment:
It took me a minute to understand this. I think it wouldve been clear quickly if this second strat said explicit weapons bombs

Reply:
That's how I had it at first -- I can put it back for clarity